src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -630,4 +630,8 @@
   <data name="Change_to_cast" xml:space="preserve">
     <value>Change to cast</value>
   </data>
+  <data name="Make_container_async" xml:space="preserve">
+    <value>Make container 'async'</value>
```

Comment:
"Make containing method 'async'"?

Reply:
@genlu I intentionally used container as it could be method, local function, lambda

Reply:
I'm ok with this.  it's not great, but it's not bad either.